Src/Support/Google.Apis.Auth.Tests/OAuth2/ComputeCredentialTests.cs

```diff
@@ -42,6 +43,19 @@ public void IsRunningOnComputeEngine_ResultIsCached()
                 ComputeCredential.IsRunningOnComputeEngine());
         }
 
+        [Fact]
+        public void WithClientFactory()
```

Comment:
Maybe rename the test to WithHttpClientFactory to match the method we're testing? (Ditto in other tests)